src/templates/AppLayout/AppLayout.tsx

```diff
@@ -0,0 +1,87 @@
+import { type ReactNode } from "react";
+import { Box, Text } from "~/components";
+import { isReactText } from "~/utils";
+import { appLayoutBoxRecipe, appLayoutTextRecipe } from "./AppLayout.css";
+
+export const AppLayout = ({
+  title,
+  description,
+  children,
+}: {
+  title: string;
+  description?: ReactNode;
+  children: ReactNode;
+}) => {
+  return (
+    <Box
+      display="flex"
+      flexDirection="column"
+      rowGap={9}
+      marginX="auto"
+      __maxWidth={1156}
+      width="100%"
+      paddingTop={10}
+      __paddingBottom="20rem"
```

Comment:
I think all spacing should be configurable, but provide some defaults, maybe with variants? Like "dense", "full-width" etc